### PR TITLE
Yaw issue51 create event budget

### LINKED ISSUE #51: Create Event Budget Page
Page Url: /entity/event-budget/event/{eventId}/add

Module: Event - Budget

Authority
- CC Head
- Administrator
- Event Head

Usage
Add event budget to event

TODO
- [x] Development

### LINKED ISSUE #53: Delete Event Budget Page
Page Url: /entity/event-budget/event/{eventId}/{id}/delete

Module: Event - Budget

Authority
- CC Head
- Administrator
- Event Head

Usage
Delete Event Budget of event

TODO
- [x] Development

### PR DESCRIPTION
**Pull request by HowardYaw**
Checkout from #55 
- resolved #51 
- resolved #53

### CODE REVIEW

**Review by kachun333 — DISMISSED** [review]

File: src/main/webapp/app/entities/budget/budget-create.tsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+                  <Button tag={Link} id="cancel-save" to={`/entity/event-budget/event/${eventId}`} replace color="cancel">
+                    <FontAwesomeIcon icon="arrow-left" />
+                    &nbsp;
+                    <Translate contentKey="entity.action.cancel">Cancel</Translate>
+                  </Button>
+                  &nbsp;
+                  <Button color="action" id="save-entity" type="submit" disabled={updating}>
+                    <FontAwesomeIcon icon="save" />
+                    &nbsp;
+                    <Translate contentKey="entity.action.create">Create</Translate>
```
Comment (junzai97) [inline src/main/webapp/app/entities/budget/budget-create.tsx:133]: `entity.action.create` has not created in the `src/main/webapp/i18n/en/global.json` yet

File: src/main/webapp/app/entities/budget/budget-create.tsx [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+                <AvGroup>
+                  <Label id="amountLabel" for="budget-amount">
+                    <Translate contentKey="clubmanagementApp.eventBudget.amount">Amount</Translate>
+                  </Label>
+                  <AvField
+                    id="budget-amount"
+                    type="number"
+                    name="amount"
+                    validate={{
+                      required: { value: true, errorMessage: 'Please Enter an amount for this budget' },
```
Comment (junzai97) [inline src/main/webapp/app/entities/budget/budget-create.tsx:76]: e in `Enter` should be lower capital.

File: src/main/webapp/app/entities/budget/budget-create.tsx [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+                  <Label id="nameLabel" for="budget-name">
+                    <Translate contentKey="clubmanagementApp.eventBudget.name">Name</Translate>
+                  </Label>
+                  <AvField
+                    id="budget-name"
+                    type="text"
+                    name="name"
+                    required
+                    validate={{
+                      required: { value: true, errorMessage: 'Please Enter a name for this budget' },
```
Comment (junzai97) [inline src/main/webapp/app/entities/budget/budget-create.tsx:92]: e in Enter should be lower capital.

File: src/main/webapp/app/entities/budget/budget-create.tsx (hunk 2 above)
Comment (junzai97) [inline src/main/webapp/app/entities/budget/budget-create.tsx]: e in Enter should be lower capital.

File: src/main/webapp/app/entities/budget/budget-create.tsx (hunk 3 above)
Comment (junzai97) [inline src/main/webapp/app/entities/budget/budget-create.tsx]: e in Enter should be lower capital.

File: src/main/webapp/app/entities/budget/budget-create.tsx (hunk 1 above)
Comment (junzai97) [inline src/main/webapp/app/entities/budget/budget-create.tsx:133]: entity.action.create has not created in the src/main/webapp/i18n/en/global.json yet

Comment (HowardYaw) [reply src/main/webapp/app/entities/budget/budget-create.tsx:133]: the content is added in #45

File: src/main/webapp/app/entities/budget/budget-create.tsx (hunk 3 above)
Comment (HowardYaw) [reply src/main/webapp/app/entities/budget/budget-create.tsx]: resolved in 948ee18a02eeba1d00248750fb41673c50daa860

File: src/main/webapp/app/entities/budget/budget-create.tsx (hunk 2 above)
Comment (HowardYaw) [reply src/main/webapp/app/entities/budget/budget-create.tsx]: resolved in 948ee18a02eeba1d00248750fb41673c50daa860

**Review by junzai97 — APPROVED** [review]

**Review by kachun333 — APPROVED** [review]

**Pull request merged**